internal/controller/apiextensions/definition/handlers.go

```diff
@@ -35,90 +33,101 @@ import (
 	v1 "github.com/crossplane/crossplane/v2/apis/apiextensions/v1"
 )
 
-// EnqueueForCompositionRevision enqueues reconciles for all XRs that will use a
-// newly created CompositionRevision.
-func EnqueueForCompositionRevision(of schema.GroupVersionKind, s composite.Schema, c client.Reader, log logging.Logger) handler.Funcs {
-	return handler.Funcs{
-		CreateFunc: func(ctx context.Context, e kevent.CreateEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
-			rev, ok := e.Object.(*v1.CompositionRevision)
-			if !ok {
-				return
+// CompositionRevisionMapFunc returns a MapFunc that maps CompositionRevisions to affected XRs.
+func CompositionRevisionMapFunc(of schema.GroupVersionKind, s composite.Schema, c client.Reader, log logging.Logger) handler.MapFunc {
```

Comment:
Note: There shouldn't be any real logic changes anywhere in this file. It's just porting the existing handlers to map funcs.